bson/src/main/org/bson/codecs/jsr310/MonthCodec.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.bson.codecs.jsr310;
+
+import org.bson.BsonReader;
+import org.bson.BsonType;
+import org.bson.BsonWriter;
+import org.bson.codecs.Codec;
+import org.bson.codecs.DecoderContext;
+import org.bson.codecs.EncoderContext;
+
+import java.time.Month;
+
+import static org.bson.codecs.jsr310.Jsr310DecoderHelper.validateIsNumeric;
+import static org.bson.internal.NumberCodecHelper.decodeInt;
+
+/**
+ * Month Codec.
+ *
+ * <p>
+ * Encodes and decodes {@code Month} objects to and from {@code Int32}.
+ * The value stored represents the nanoseconds of the day.
```

Comment:
This line is probably copied by mistake ;)

Reply:
Done.